Server/htdocs/AppController/commands_RSM/utilities/RSMverifyBody.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+//***************************************************************************************
+// Structure
+//***************************************************************************************
+
+function checkBodyIsJsonObject($body)
+{
+  global $RSallowDebug;
+  if (!is_object($body)) {
+    if ($RSallowDebug) returnJsonMessage(400, "Request body must be a JSON object '{}'");
+    else returnJsonMessage(400, "");
+  }
+}
+
+function checkIsJsonObjectInArray($body)
+{
+  global $RSallowDebug;
+  if (!is_object($body)) {
+    if ($RSallowDebug) returnJsonMessage(400, "Request body array elements must be JSON objects '{}'");
+    else returnJsonMessage(400, "");
+  }
+}
```

Comment:
This function is the same as the one before, only changes the error message. There should be a validation ob "object is array" in this one.